src/EnergyPlus/MundtSimMgr.cc

```diff
@@ -297,12 +293,12 @@ namespace MundtSimMgr {
                         }
 
                         // count air nodes connected to walls in each zone
-                        if (state.dataMundtSimMgr->LineNode(NodeNum, MundtZoneIndex).ClassType == MundtRoomAirNode) {
+                        if (state.dataMundtSimMgr->LineNode(NodeNum, MundtZoneIndex).ClassType == DataRoomAirModel::AirNodeType::MundtRoomAirNode) {
                             ++RoomNodesCount;
```

Comment:
Same as before, probably don't need AirNode in the enumerated values themselves if using scoped enum.